src/Components/Form/FormFields/AutoExpandingTextInputFormField.tsx

```diff
@@ -0,0 +1,24 @@
+import React, { useEffect, useRef } from "react";
+import TextAreaFormField, { TextAreaFormFieldProps } from "./TextAreaFormField";
+
+type AutoExpandingTextInputFormFieldProps = TextAreaFormFieldProps & {
+  maxHeight?: number;
+};
+
+const AutoExpandingTextInputFormField = (
+  props: AutoExpandingTextInputFormFieldProps
+) => {
+  const myref = useRef<HTMLTextAreaElement>(null);
+  useEffect(() => {
+    if (myref.current == null) return;
+    const text = myref.current.textContent?.split("\n");
+    const len = (text?.length == 0 ? 1 : text?.length) || 1;
+    const height = Math.min(len * 18, (props.maxHeight || 160) - 28) + 28;
+    myref.current.style.cssText =
+      "min-height:36px; height:46px; height:" + height + "px";
```

Comment:
`min-height:36px;` is constant here, move it to the tailwind className of TextAreaFormField.

`height:46px; height:" + height + "px";`
You are setting height to 46px then again adding a separate height override afterwards again. Why so?